dbt_opiner/dbt.py

```diff
@@ -307,7 +307,10 @@ def compiled_code(self) -> str:
 
     @property
     def docs_yml_file_path(self) -> str:
-        return self._node.get("patch_path", "").replace("://", "/")
+        patch_path = self._node.get("patch_path", "")
+        if patch_path:
+            return patch_path.replace("://", "/")
+        return ""
```

Comment:
return patch_path.replace("://", "/") if patch_path is not None else ""
```suggestion
        if patch_path:
            return patch_path.replace("://", "/")
        return ""
```